<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,2">
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Y Corredinate</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Water Level</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="Valve">
        <property name="text">
         <string>Valve</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="12" column="0">
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>X Corredinate</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="cityComboBox"/>
      </item>
      <item row="7" column="0">
       <widget class="QLineEdit" name="Y_coordinate"/>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="submitButton">
        <property name="text">
         <string>Get Weather prediciton</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="X_coordinate"/>
      </item>
      <item row="11" column="0">
       <widget class="QPushButton" name="pushButton_Measure_distance">
        <property name="text">
         <string>Measure Distance</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QTextEdit" name="textEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="stateComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Cumulative">
        <property name="text">
         <string>Raining Amount perdiciton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="LayoutProbPanel">
      <item>
       <widget class="QListWidget" name="listWidgetProb"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="LayoutQuanPanel">
      <item>
       <widget class="QListWidget" name="listWidgetQuan"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
